### ISSUE TITLE
Prerelease flag not working, probably a wrong api call to semver

### ISSUE THREAD

**Issue by PorterNan** [microsoft/beachball#632]
I found out --prerelease-prefix=beta not working well in current version, so I checked the code in runtime and found probably a wrong api call to semver.inc:

![image](https://user-images.githubusercontent.com/11863655/151643966-0da49b89-0187-496f-8665-cf384ab997bc.png)
And this is not working because semver.inc changed their api call:
Beachball is calling it like:
```
semver.inc(version, 'patch' | 'minor', 'beta')
```

According to what semver doc said, It should be like this:
```
semver.inc(version, 'prelease', 'beta')
```
Wish there could be a logic fix for this and make prerelease bump working

Correct call (1st line) and possible a wrong call(3rd line)
![image](https://user-images.githubusercontent.com/11863655/151644156-58004514-5cab-40f6-964d-cd9c63774392.png)

**Comment by PorterNan** [comment]
Open a PR on this issue for prerelease #633

**Comment by awentzel** [comment]
Hi @PorterNan how did you work around this issue?

**Comment by ecraig12345** [comment]
Looks like this was fixed by #658
